cmd/flags/flags.go

```diff
@@ -101,4 +102,8 @@ var (
 		Usage: "",
 		Value: false,
 	}
+	TokenId = &cli.StringFlag{
```

Comment:
We should add it into server cmd to be able to pre-set API key in development mode (--in-memory). Skip it in production mode (if backend is set).